src/ts/components/Results.tsx

```diff
@@ -34,8 +34,12 @@ interface IProps {
     getScrollOffset
   }: IProps) => {
 
+    const IMPORTANCE = "importance";
+    const APPEARANCE = "appearance";
```

Comment:
Nit: I'd always put anything that's constant for each invocation outside of the function body. The performance difference in this case will be undetectable, so this is purely aesthetic and you're free to ignore!